src/main/java/life/qbic/model/download/QbicDataDownloader.java

```diff
@@ -61,23 +52,23 @@ public class QbicDataDownloader {
    *
    * @param AppServerUri  The openBIS application server URL (AS)
    * @param DataServerUri The openBIS datastore server URL (DSS)
-   * @param user          The openBIS user
-   * @param password      The openBis password
+   //* @param user          The openBIS user
+   //* @param password      The openBis password
    * @param bufferSize    The buffer size for the InputStream reader
```

Comment:
```suggestion
   * @param AppServerUri  The openBIS application server URL (AS)
   * @param DataServerUri The openBIS datastore server URL (DSS)
   * @param bufferSize    The buffer size for the InputStream reader
   * @param filterType //I think you can remove this one
   * @param conservePaths // TODO document
   * @param checksumReporter // TODO document
   * @param sessionToken The session token for the openBisApp (or is it the data server?)
   ```

Reply:
Oh right I forgot to edit this. 

But I have two questions considering the options:
- do you know for what we need the conserve Paths option? Because I used it and did not notice any change... it is used in the code but I can't really tell what the goal here is
- The filter and the regex option are currently not used and the suffix one only for the download... I think it would not be that much work to make them work again but I was wondering if there is still a need for them.